packages/terra-dropdown-button/src/_DropdownListUtil.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
```

Comment:
nit: There are a few references to `menu` in the comment documentation in this file. They could be cleaned up.

Reply:
Done in f3b73577383099a9bddb5383f6a820868611af97